STM32Cube/Core/Inc/FreeRTOSConfig.h

```diff
@@ -68,7 +68,7 @@
 #define configTICK_RATE_HZ                       ((TickType_t)1000)
 #define configMAX_PRIORITIES                     ( 56 )
 #define configMINIMAL_STACK_SIZE                 ((uint16_t)128)
-#define configTOTAL_HEAP_SIZE                    ((size_t)15360)
+#define configTOTAL_HEAP_SIZE                    ((size_t)131072)
```

Comment:
it looks like you changed this in the source file and not the .ioc
When the ioc regenerates this will be overwritten

Reply:
updated ioc woohoo i love generated code